### DISCUSSION TITLE
`supabase start` service role and anonymous keys never change

### DISCUSSION THREAD

**Discussion by ValentaTomas** [supabase/supabase#13377] [Questions] [answered]
After playing with the local Supabase development it looks like the `service_role` and `anon` keys are the same every time I start the Supabase with `supabase start`.

Can I rely on the keys to stay the same? Are there any security implications?

**Comment by silentworks** [comment]
As `supabase start` is purely for local development, you don't need to worry about these remaining the same.

**Reply by ValentaTomas** [reply]
Can I also rely on them being the same? 
I want to commit them in a `.env.example` used for local development of a project so people don't have to copy the vars from `supabase start`.

**Reply by silentworks** [reply, marked as answer]
Yes they remain the same, I commit mine in an `.env.example` file in my repos and note that it should be renamed to `.env`.

**Reply by ValentaTomas** [reply]
Awesome

**Reply by softmarshmallow** [reply]
@silentworks would you say that as recommend pattern for keeping .env.example for other OSS projects for faster onboarding?
(will it NEVER change?)

**Reply by aantti** [reply]
@jgoux, @avallete fyi
